dev/deps/spark-deps-hadoop-2.7

```diff
@@ -112,6 +113,7 @@ jets3t-0.9.3.jar
 jetty-6.1.26.jar
 jetty-util-6.1.26.jar
 jline-2.12.1.jar
+jna-4.2.2.jar
```

Comment:
Hope this one doesn't add problems... is it really required by the crypto library? Or could we exclude it (and the library falls back to JCE if JNI libraries are not available)?